vterm.el

```diff
@@ -276,6 +256,49 @@ If nil, never delay")
                                        unless (member key vterm-keymap-exceptions)
                                        collect key))))
 
+;; Keybindings
+(define-key vterm-mode-map [tab]                       #'vterm-send-tab)
+(define-key vterm-mode-map (kbd "TAB")                 #'vterm-send-tab)
+(define-key vterm-mode-map [backtab]                   #'vterm--self-insert)
+(define-key vterm-mode-map [backspace]                 #'vterm-send-backspace)
```

Comment:
Why did you change these? For terminal support?

Reply:
yes